js/templates/miniProfile.html

```diff
@@ -1,9 +1,12 @@
 <div class="flexVCent">
   <div>
-    <div class="userIcon disc clrBr2 clrSh1" style="<%= ob.getAvatarBgImage(ob.avatarHashes || {}) %>"></div>
+    <% const userLink = ob.peerIDx ? `href="#${ob.peerID}"` : ''; %>
```

Comment:
@jjeffryes Does the template really give you a variable named `peerIDx`? Is that coming from the server? I'm not seeing where that is coming from.